permissions/CallLogs.permission

```diff
@@ -0,0 +1,14 @@
+# -*- mode: sh -*-
+
+# x-sailjail-translation-catalog = sailjail-permissions
+# x-sailjail-translation-key-description = permission-la-calllogs
+# x-sailjail-description = Call history
```

Comment:
While this is also about message history, I used word call here. I wonder if there was some nice way of combining them. "Call and message history" is a bit long IMO and translations can be even longer.

Also that could affect permission file naming if we come up with something better here.

Reply:
Well, commhistory = "Communications event history" -> maybe there would be something not too clumsy / abstract in that direction? But I'm not sure if  "CommunicationHistory" fits that bill.

Reply:
Yes, that's even longer, at least in English. Well, I didn't change this quite yet to anything else. And perhaps that call history is still understandable enough with that longer explanation.

Reply:
It seems there should be quite enough space for "communication history" so I'll use that for now. Changed that file name too.